packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+                "version": "12ba4e7e158aa0b78d78c7e9ae15d917"
+            }
+        }]
+    }
+
+**Note:** The `itemsPerPage`, `startIndex`, and `totalResults` values need to be returned as integers and not as strings.
+
+### Retrieving specific groups
+
+<ApiOperation method="get" url="/groups/${groupId}" />
+
+There are situations where Okta needs to run a GET operation on a specific `${groupId}`, for example to see if the group still exists on the SCIM server. The request looks like the following:
+
+    GET /scim/v1/groups/74094a55-c9ee-47ae-9fd4-9137deb43497 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+The SCIM server response is to return the group details:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:08:51 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:02:44+00:00",
+            "version": "12ba4e7e158aa0b78d78c7e9ae15d917"
+        }
+    }
+
+### Update specific group name
+
+<ApiOperation method="patch" url="/groups/${groupId}" />
+
+When a group is pushed to the SCIM server, most of the time the group name is updated or inserted by Okta through a PATCH operation:
+
+    PATCH /scim/v1/groups/74094a55-c9ee-47ae-9fd4-9137deb43497 HTTP/1.1
+    User-Agent: Okta SCIM Client 1.0.0
+    Authorization: <Authorization credentials>
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2"
+    }
+
+The SCIM server response is to return the group details, just as for a group retrieval request:
+
+    HTTP/1.1 200 OK
+    Date: Fri, 18 Oct 2019 07:11:00 GMT
+    Content-Type: text/json;charset=UTF-8
+
+    {
+        "schemas": ["urn:scim:schemas:core:1.0"],
+        "id": "74094a55-c9ee-47ae-9fd4-9137deb43497",
+        "displayName": "Test SCIM2",
+        "members": [],
+        "meta": {
+            "created": "2019-10-18T07:02:44+00:00",
+            "lastModified": "2019-10-18T07:11:00+00:00",
+            "version": "3a0c8354ccfcaab56019859469078a0b"
+        }
+    }
+
+### Update specific group membership
+
+<ApiOperation method="patch" url="/groups/${groupId}" />
+
+To add users to a specific pushed group on the SCIM server, Okta requires:
+
+* The user must be a member of the group in Okta
+* The user has been added under the **Assignments** tab of the SCIM application inside the Okta Admin Console
+* The group is pushed under the **Push Groups** tab of the SCIM application inside the Okta Admin Console
+
+If these three requirements are met; Okta sends a PATCH request to add a user on the SCIM server. The request looks similar to the following:
```

Comment:
If these three requirements are met, Okta sends a PATCH request to add a user on the SCIM server. The request looks similar to the following: